rfcs/20191101-embedding-variable.md

```diff
@@ -0,0 +1,195 @@
+# Embedding Variable in TensorFlow
+
+
+| Status        | Proposed                |
+| ------------- | ----------------------- |
+| **Author(s)** | dingchen.mail@gmail.com |
+| **Sponsor**   |                         |
+| **Updated**   | 2019-11-01              |
+
+## Problem & Motivation
+
+"In TensorFlow, we model all data as tensors(n-dimensional arrays) …... At the lowest level, all TensorFlow tensors are dense…... " which quote from TensorFlow white paper<sup>[1]</sup>  states that Tensor is the basic data structure in TensorFlow. Every component(Operation, Rendervous, BuddleReader, BuddleWriter etc.) in TF framework depends on Tensor. It works well with simple interface, clear hierarchy, elegent design.
+
+​    In large scale sparse data training, `embedding_lookup` is the first layer in train graph, the data storage and transmission is rigid. Usually, users invoke the API and build graph as below:
+
+```python
+...
+col0 = tf.contrib.layers.sparse_column_with_hash_bucket(column_name = "col0",
```

Comment:
`tf.contrib` has been remove from core, maybe some changes needed.

Reply:
yes, the rfc will be update soon